### PR TITLE
Develop ben

### PR DESCRIPTION
**Pull request by benwade91**
styled the NoMatch page

### CODE REVIEW

File: client/src/pages/NoMatch/index.jsx [hunk 1]
```diff
@@ -1,21 +1,38 @@
 import React from 'react';
 import Nav from '../../components/Nav';
 import noParking from '../../images/NoParking.png';
+import bang from './gifs/bang.gif';
+import bounce from './gifs/bounce.gif';
+import crash from './gifs/crash.gif';
+import ouch from './gifs/ouch.gif';
+import whoops from './gifs/whoops.gif';
```
Comment (yulduzetta) [inline client/src/pages/NoMatch/index.jsx:9]: good job on creative file names

File: client/src/components/Map/index.jsx [hunk 2]
```diff
@@ -53,25 +43,25 @@ function MyMapComponent() {
...
-            </div>
+            </div> : null}
+            
             <GoogleMap
                 key={new Date().getTime()}
                 mapContainerStyle={containerStyle}
                 zoom={15}
-                center={state.mapLocation}
+                center={state.mapLocation ? state.mapLocation : { lat: 37.774, lng: -122.419 }}
                 options={options}
```
Comment (yulduzetta) [inline client/src/components/Map/index.jsx:65]: :100:

File: client/src/components/Map/index.jsx [hunk 3]
```diff
@@ -1,50 +1,40 @@
...
-    //     id: 'google-map-script',
-    //     googleMapsApiKey: process.env.REACT_APP_GOOGLE_API,
-    //     libraries
-    // })
+function MyMapComponent(props) {
+
+    // const { loading, data } = useQuery(QUERY_ALL_PARKING);
+
+    // console.log(loading);
+    // console.log(data);
```
Comment (yulduzetta) [inline client/src/components/Map/index.jsx:33]: do we need this unused code?

File: client/src/pages/NoMatch/index.jsx [hunk 4]
```diff
@@ -1,21 +1,38 @@
...
 
 import './style.scss';
 
 const NoMatch = () => {
-    window.setTimeout( function(){
-        window.location = "/";
-    }, 4000 );
+    // window.setTimeout(function () {
+    //     window.location = "/";
+    // }, 10000);
```
Comment (yulduzetta) [inline client/src/pages/NoMatch/index.jsx:15]: so we don't need the auto-redirect anymore or will we add it back later?

File: client/src/components/Map/index.jsx (hunk 3 above)
Comment (benwade91) [reply client/src/components/Map/index.jsx:33]: it throws an error at the moment when uncommented but its going to be used to query the parking spots

File: client/src/pages/NoMatch/index.jsx (hunk 4 above)
Comment (benwade91) [reply client/src/pages/NoMatch/index.jsx:15]: it'll be back in the next PR. forgot to uncomment when i pushed

**Review by yulduzetta — APPROVED** [review]
looks super awesome to me --gtg 🚢

**Pull request merged**